docs/robots/jackal/integration_jackal.mdx

```diff
@@ -88,6 +88,16 @@ pin does not exceed 8 A.
 
 :::
 
+#### MCU Debug LEDs
+
+The Jackal MCU PCBA comes equipped with four LEDs controlled by Jackal firmware for debugging.
```

Comment:
I suggest this be changed to _Jackal MCU circuit board_ to reduce the number of acronyms in this sentence.

Reply:
Fixed